### PR TITLE
Release notes for 7.5

### PR DESCRIPTION
**Pull request by rymsha**

### CODE REVIEW

**Comment by sigdestad** [comment]
Looks good, but did we also document these new features in NoQL section and usage example?

File: docs/release.adoc [hunk 1]
```diff
@@ -4,6 +4,29 @@ include::variables.adoc[]
...
+
+`latest` option simplifies day-to-day snapshot/restore rutines.
+
+=== Meta-data in contentLib query results
+
+_sort and _score meta fields are exposed in contentLib query results. Now it is possible to know how exacly content items were sorted.
+
+=== Calculated distance in geoDistance aggregation
+
+Another use of _sort mate field is a geoDistance value.
```
Comment (alansemenov) [inline docs/release.adoc:19]: _mate field_?

**Comment by rymsha** [comment]
#155 #179 document new NoQL features

**Review by jsi — APPROVED** [review]
Nice!

**Pull request merged**